docs/fr-FR/flashing.md

```diff
@@ -99,11 +99,14 @@ ou, utilisez :
 
     make <keyboard>:<keymap>:avrdude
 
-ou, si vous vous voulez flasher plusieurs claviers, utilisez la commande suivante :
+#### Commandes Caterina 
 
-    make <keyboard>:<keymap>:avrdude-loop
+Il existe un certain nombre de commandes DFU que vous pouvez utiliser pour mettre à jour le micrologiciel sur un périphérique DFU:
```

Comment:
In the translated parts, we haven't translated firmware to "micrologiciel", so to keep consistency we should either translated firmware to micrologiciel everywhere or keep firmware here.

Reply:
Gimly, like this? 
```suggestion
Il existe un certain nombre de commandes DFU que vous pouvez utiliser pour mettre à jour le firmware sur un périphérique DFU:
```

Reply:
yes